src/main/webapp/download.html

```diff
@@ -417,18 +397,13 @@ <h3 class="mb-0 text-dark">Karaf Cellar <span class="text-muted">4.0.4</span></h
                         <div class="card-body d-flex flex-column align-items-start">
                           <strong class="d-inline-block mb-2 text-warning"><i class="fas fa-circle"></i> Latest release</strong>
                           <h3 class="mb-0 text-dark">Karaf Cellar <span class="text-muted">3.0.3</span></h3>
-                          <div class="mb-1 text-muted">April 9, 2018</div>
-                          <p class="card-text mb-auto">
-                            Binary Distribution :
-                              <a href="http://www.apache.org/dyn/closer.lua/karaf/4.2.0/apache-karaf-4.2.0.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0.tar.gz.sha512">SHA512</a>] - 
-                                            <a href="http://www.apache.org/dyn/closer.lua/karaf/4.2.0/apache-karaf-4.2.0.zip">zip</a> [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0.zip.asc">PGP</a>] [<a href="https://www.apache.org/dist/4.2.0/apache-karaf-4.2.0.zip.sha512">SHA512</a>]
-                          </p>
+                          <div class="mb-1 text-muted">April 4, 2016</div>
                           <p class="card-text mb-auto">
                             Source Distribution :
-                              <a href="http://www.apache.org/dyn/closer.lua/karaf/4.2.0/apache-karaf-4.2.0-src.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0-src.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0-src.tar.gz.sha512">SHA512</a>] - 
-                                            <a href="http://www.apache.org/dyn/closer.lua/karaf/4.2.0/apache-karaf-4.2.0-src.zip">zip</a> [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0-src.zip.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/4.2.0/apache-karaf-4.2.0-src.zip.sha512">SHA512</a>]
+                              <a href="http://www.apache.org/dyn/closer.lua/karaf/cellar/3.0.3/apache-karaf-cellar-3.0.3-src.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/cellar/3.0.3/apache-karaf-cellar-3.0.3-src.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/cellar/3.0.3/apache-karaf-cellar-3.0.3-src.tar.gz.sha1">SHA1</a>] - 
```

Comment:
I think it should go to `archive.apache.org` instead of the main mirrors.

Reply:
Ok, I made change on it